src/core/images.js

```diff
@@ -0,0 +1,21 @@
+// Module core/images
+// This checks for images missing width and height attributes
+// and adds them if they are missing
+
+import { pub } from "core/pubsubhub";
+
+export const name = "core/images";
+
+export function run(conf, doc, cb) {
+  [...doc.querySelectorAll("img")]
+    .filter(img => !img.hasAttribute("width") || !img.hasAttribute("height"))
```

Comment:
WE should probably just do "&&" here. If someone has set one, but not the other, they probably have a good reason. 

We should also filter for `srcset` being set - and if `img` is inside a `<picture>` element, we should probably filter it out too.